iob_soc_opencryptolinux.py

```diff
@@ -121,6 +139,11 @@ def _create_submodules_list(cls, extra_submodules=[]):
 
     @classmethod
     def _post_setup(cls):
+        # OpenCrypts testcases
+        shutil.copytree(
+            f"{cls.setup_dir}/tests", f"{cls.build_dir}/tests", dirs_exist_ok=True
+        )
```

Comment:
should we just place tests under `./software/src/tests` and they are automatically copied to `BUILD_DIR`?

or add to `./software/versat/tests` and add `versatSpec.txt` to `./software/versat/versatSpec.txt` for versat-specific files